### ISSUE TITLE
pip install not working for me

### ISSUE THREAD

**Issue by focus-shift** [simpeg/simpeg#173]
Running sudo pip install SimPEG currently fails for me. I'm running Kubuntu 14.04. Running the install command gives

Downloading/unpacking SimPEG
  Downloading SimPEG-0.1.3.tar.gz (154kB): 154kB downloaded
  Running setup.py (path:/tmp/pip_build_root/SimPEG/setup.py) egg_info for package SimPEG
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/SimPEG/setup.py", line 53, in <module>
        ext_modules = cythonize('SimPEG/Utils/interputils_cython.pyx')
      File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 758, in cythonize
        aliases=aliases)
      File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 651, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 103, in nonempty
        raise ValueError(error_msg)
    ValueError: 'SimPEG/Utils/interputils_cython.pyx' doesn't match any files
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/SimPEG/setup.py", line 53, in <module>

    ext_modules = cythonize('SimPEG/Utils/interputils_cython.pyx')

  File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 758, in cythonize

    aliases=aliases)

  File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 651, in create_extension_list

    for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):

  File "/usr/local/lib/python2.7/dist-packages/Cython/Build/Dependencies.py", line 103, in nonempty

    raise ValueError(error_msg)

ValueError: 'SimPEG/Utils/interputils_cython.pyx' doesn't match any files

I have cython successfully installed (via pip).

**Comment by bsmithyman** [comment]
Thanks @focus-shift,

I believe this is the same issue that cropped up in #116. We just need to upload a new `SimPEG-0.1.3.tar.gz` to PyPi. Quick fix in the short term is to skip PyPi and install from source with pip.

Current Git master:
```shell
pip install git+https://github.com/simpeg/simpeg.git
```

Same release as PyPi:
```shell
pip install git+https://github.com/simpeg/simpeg.git@v0.1.3
```

**Comment by focus-shift** [comment]
Thanks very much for the quick reply @bsmithyman. Installing from source using
```
pip install git+https://github.com/simpeg/simpeg.git@v0.1.3
```
worked for me. 

Patrick

**Comment by bsmithyman** [comment]
No worries, glad to help. Talked to @rowanc1, and he's planning to upload the new archive to PyPi for regular pip this afternoon.

Brendan

**Comment by rowanc1** [comment]
This should now be fixed on Pip! Use: v0.1.9
Sorry for the delay!
-Rowan
